calico_containers/pycalico/datastore.py

```diff
@@ -129,7 +137,15 @@ def ensure_global_config(self):
 
         # Configure IPAM directory structures (to ensure confd is able to
         # watch appropriate directory trees).
-        self.etcd_client.write(CONFIG_IF_PREF_PATH, IF_PREFIX)
+        host = socket.gethostname()
```

Comment:
Comment was wrong, but I'm guessing we still need the 
```
self.etcd_client.write(CONFIG_IF_PREF_PATH, IF_PREFIX)
```

What this does is configure Felix to look for the right interface prefix for container veths.